python/pyspark/pandas/tests/data_type_ops/test_categorical_ops.py

```diff
@@ -46,24 +46,20 @@ def other_psser(self):
 
     @property
     def ordered_pser(self):
-        return pd.Series([1, 2, 3]).astype(CategoricalDtype([3, 2, 1], ordered=True))
+        return pd.Series([1, "x", "y"]).astype(CategoricalDtype(["y", 1, "x"], ordered=True))
```

Comment:
I'm not sure mixing integer and string in the categories is valid for us?

Reply:
Makes sense. Let me separate that.

Reply:
The example in the PR description is updated as well.